src/olympia/blocklist/mlbf.py

```diff
@@ -384,6 +398,38 @@ def should_upload_stash(
             > 0
         )
 
+    def validate(self):
+        store: Dict[str, Dict[MLBFDataType, int]] = {}
+
+        # Create a map of each guid:version string in the cache.json
+        # and the set of data types that contain it and the count of
+        # how many times it occurs in each data type.
+        for key in MLBFDataType:
+            for item in self.data[key]:
+                if item in store:
+                    if key in store[item]:
+                        store[item][key] = store[item][key] + 1
+                    else:
+                        store[item][key] = 1
+                else:
+                    store[item] = {key: 1}
+
+        # Verify that each item occurs only one time and in only one data type
+        for item, data_types in store.items():
+            # We expect each item to only occur in one data type
+            if len(data_types) > 1:
+                formatted_data_types = ', '.join(key.name for key in data_types.keys())
+                raise ValueError(
+                    f'Item {item} found in multiple data types: '
+                    f'{formatted_data_types}'
```

Comment:
nit
```suggestion
                    f'Item {item} found in multiple data types: {formatted_data_types}'
```

Reply:
I think that fails linting for line to long but will try.